st2client/st2client/models/core.py

```diff
@@ -195,6 +196,9 @@ def get_all(self, **kwargs):
         if user:
             params["user"] = user
 
+        if offset:
```

Comment:
Since if we use a truthy check, this means ``offset`` won't be included if set to ``0``, but yeah I think this should be fine since that's the current / existing default behavior if ``offset`` is not specified.